app/qml/components/MMCircleIcon.qml

```diff
@@ -0,0 +1,47 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+
+Rectangle {
+  id: root
+
+  enum Type {Edit, Checkbox, Close, SmallCheckmark}
+
+  required property int type
+  required property url source
+  required property real size
+  property color iconColor: __style.forestColor
+
+  color: __style.grassColor
+  width: size
+  height: size
+  radius: size / 2
+
+  MMIcon {
```

Comment:
Do not use MMIcon here - use Image{} directly

Reply:
better to use MMIcon due to the coloring icon in the middle, I think